src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -984,34 +984,76 @@ def is_valid_uint_value(value, bit_count=64):
             return False
         return value < 2**bit_count
 
+    @staticmethod
+    def is_valid_int_value(value, bit_count=8):
+        """
+        Checks if 'value' is a signed integer that fits into 'bit_count' bits.
+        For example, for int8: -128 <= value <= 127.
+        """
+        min_val = -(2 ** (bit_count - 1))
+        max_val = (2 ** (bit_count - 1)) - 1
+        return isinstance(value, int) and (min_val <= value <= max_val)
+
+    @staticmethod
+    def is_valid_bool_value(value):
```

Comment:
can we move most of these into https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/matter_testing_infrastructure/chip/testing/matter_asserts.py and start using those?